custom/ilsgateway/api.py

```diff
@@ -472,6 +472,9 @@ def sms_user_sync(self, ilsgateway_smsuser, **kwargs):
         if not sms_user:
             return None
 
+        if not sms_user.is_active:
```

Comment:
Rest of logic is done in superclass: https://github.com/dimagi/commcare-hq/pull/10490/files#diff-a9531de75432f1a2708aeeeaffa9ea49R325